### PR TITLE
Makerspace Link added & Team Page Updated

### PR DESCRIPTION
**Pull request by kushal2000**

### CODE REVIEW

File: _data/alumni.yml [hunk 1]
```diff
@@ -1,10 +1,97 @@
...
+      name: "Vraj"
+      surname: "Parikh"
+      thumbnailUrl: vraj.jpg
+      team: true
+      social:
+        - {name: "site", link: "http://www.vrajparikh.in/"}
+        - {name: "github", link: "https://www.github.com/vrjparikh"}
+        - {name: "email", link: "mailto:vraj@robotix.in"}
+    -
+      id: 38
```
Comment (shivangag) [inline _data/alumni.yml]: @kushal2000 Please, sort the names of my batch alphabetically.

File: _config.yml [hunk 2]
```diff
@@ -46,6 +46,7 @@ socialImage: "/img/favicons/favicon-196x196.png"
 
 # Navigation
 navigationLinks:
+ - {permalink: "https://makerspace.robotix.in/", text: "Makerspace"}
```
Comment (shivangag) [inline _config.yml]: Are you sure this line works? I think the final URL generated will be something like `https://www.robotix.in/https://makerspace.robotix.in/`, although I'm not sure. Please check.

Comment (kushal2000) [reply _config.yml]: It was working when I tested it locally

File: _data/alumni.yml (hunk 1 above)
Comment (kushal2000) [reply _data/alumni.yml]: Should I keep the 4th year graduates and 5th year graduates seperated or sort them together?

Comment (shivangag) [reply _data/alumni.yml]: Keep them separate.

Comment (kushal2000) [reply _data/alumni.yml]: I have pushed the alphabetical order change in another pull request

**Comment by shivangag** [comment]
@kushal2000 Good work. I'll merge this pull request.
Here are a couple of points for you to keep in mind:
* Always create separate commits for unrelated changes. For example, here it would have been more sensible to have separate commit for adding makerspace link and updating team page.
* Don't make changes and open PRs from your master branch. This helps to keep the git commit-tree of master branch same across all copies of the repository.

**Pull request merged**